nextflow.config

```diff
@@ -173,7 +173,7 @@ params {
 
     abricate {
         singularity = "https://depot.galaxyproject.org/singularity/abricate%3A1.0.1--ha8f3691_1"
-        docker = "quay.io/biocontainers/abricate:1.0.1--ha8f3691_1"
+        docker = "biocontainers/abricate:1.0.1--ha8f3691_1"
```

Comment:
If we remove the `quay.io` registry prefix we need to include the `docker.registry = 'quay.io'` 697cbb2